### PR TITLE
Allow the default values for AdvancedConfig to be changed

### PR DESCRIPTION
**Pull request by kennyt**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/js/components/SchemaForm.js [hunk 1]
```diff
@@ -118,9 +118,15 @@ class SchemaForm extends React.Component {
   }
 
   getNewDefinition() {
-    return SchemaUtil.schemaToMultipleDefinition(
+    let definition = SchemaUtil.schemaToMultipleDefinition(
       this.props.schema, this.getSubHeader, this.getLabel
     );
+
+    if (this.props.model) {
```
Comment (aldipower) [inline src/js/components/SchemaForm.js]: 'let {props} = this;` it's used twice now.

File: src/js/components/SchemaForm.js [hunk 2]
```diff
@@ -118,9 +118,15 @@ class SchemaForm extends React.Component {
...
+      SchemaFormUtil.mergeModelIntoDefinition(this.props.model, definition);
```
Comment (mlunoe) [inline src/js/components/SchemaForm.js]: 1. This function doesn't exist yet?
2. And is it supposed to modify `definition`? Not sure that's the best path to go down. maybe just return a new `definition`?

Comment (kennyt) [inline src/js/components/SchemaForm.js]: this function exists already. this function is a merge, similar to `Object.assign` or `extend`

**Comment by jfurrow** [comment]
Looks good, let's 📦 when @aldipower's comment is addressed.

**Comment by kennyt** [comment]
@jfurrow @aldipower resolved comment 👍

**Pull request merged**